vm/src/obj/objbyteinner.rs

```diff
@@ -843,6 +843,22 @@ impl PyByteInner {
             .to_vec()
     }
 
+    // new in Python 3.9
+    pub fn removeprefix(&self, prefix: PyByteInner) -> Vec<u8> {
+        if self.elements.starts_with(&prefix.elements) {
+            return self.elements[prefix.elements.len()..].to_vec();
+        }
+        self.elements.to_vec()
```

Comment:
I feel like this line cause unnessessary data copying instead of increment of reference count. But I don't have simple fix suggestion of this.